actions/testmo-create-resources/action.yml

```diff
@@ -0,0 +1,38 @@
+name: create testmo resources
+description: 'create a TestMo resource .json file'
+inputs:
+  resources_file:
+    description: "file name where the resources will be stored"
+    required: true
+  python_version:
+    description: "version of python"
+    default: ""
+    required: false
+  runner:
+    description: "runner hardware configuration label"
+    default: ""
+    required: false
+  lm_eval_configuration:
+    description: "list of configurations tested with LM-Eval"
+    default: ""
+    required: false
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        echo "composing TESTMO resources file ${{ inputs.resources_file }} ..."
+        declare -A fields=( \
```

Comment:
I think it's better to just pass in the test_config and construct the fields from the test_config so that we don't need to have separate input arguments for each field, plus we don't need python for the lm eval reporting etc.

Reply:
I'm not sure what you mean by the "test_config", but I get what you're saying about too many input args.  I'm working on an alternative.
re: the python_version is being provided because that's one of the "resources" to be added to the TestMo thread.  no python is being run based on this field.

Reply:
The "test_config" from nm-build-test.yml which contains info for python version, runner, config file.

We don't need to add python_version in the lm-eval thread, it's only needed for the test thread reporting.

Reply:
so, `nm-build-test.yml` generates multiple **LM-EVAL** jobs based on the matrix of `lm-eval-configurations` passed to it.  each LM-EVAL job gets one of those configurations.  each lm-eval-configuration is a string that looks like this (for example):
```
'{"label":"gcp-k8s-l4-solo","config":"./.github/lm-eval-configs/solo-duo-small-smoke-models.txt","timeout":"300"}'
```
the job then breaks out the specifics from this lm-eval-configuration string into args that are passed to the `nm-lm-eval.yml` workflow.  (see https://github.com/neuralmagic/nm-vllm-ent/blob/207471a9f8fb072d601f308d0c25e8e915fa4ea0/.github/workflows/nm-build-test.yml#L147-L152)

So, by the time we get to the `nm-lm-eval.yml` workflow (where I am adding the steps create a resources file and to report the thread results), there is no single "test_config".  I'm working on replacement code to re-combine the relevant "resources" into a single arg to pass to `testmo-create-resources/action.yml`.

wrt the version of python.  my notes from our conversation in standup yesterday had this as one of the "resource" attributes to include w/ the LM-Eval.  If you don't think we want it after all, I'll remove it.

Reply:
with my revisions, it's easy enough to add it back if we want it.